src/neo/VM/Helper.cs

```diff
@@ -273,20 +273,22 @@ private static ContractParameter ToParameter(StackItem item, List<(StackItem, Co
                         Value = i.ToBigInteger()
                     };
                     break;
-                case InteropInterface _:
+                case InteropInterface i:
                     parameter = new ContractParameter
                     {
-                        Type = ContractParameterType.InteropInterface
+                        Type = ContractParameterType.InteropInterface,
```

Comment:
I think that we should not be able to serialize an Interop Interface type, this type is expected to be used only inside the Application Engine.